README.md

```diff
@@ -85,6 +85,8 @@ A curated list of awesome MATLAB toolboxes, applications, software and resources
 * [Fortran mex routines](http://uk.mathworks.com/help/matlab/write-fortran-mex-files.html) - Fortran MEX routines allow you to call Fortran code from within MATLAB.
 * [MATLAB Engine for Python](http://uk.mathworks.com/help/matlab/matlab_external/get-started-with-matlab-engine-for-python.html) - The MATLAB Engine for Python provides a Python package named `matlab` that enables you to call MATLAB functions from within Python.
 * [Python](http://uk.mathworks.com/help/matlab/call-python-libraries.html) - How to call Python functions from within MATLAB.
+* [matlab kernel](https://github.com/calysto/matlab_kernel) - to run matlab code inside IPython / Jupyter notebooks.
```

Comment:
Would you mind adding capital letters where appropriate (start of sentences, "MATLAB", "Octave", ...) ?